packages/core/src/components/graphs/radar.ts

```diff
@@ -0,0 +1,473 @@
+		blobUpdate.join(
+			enter => enter
+				.append("path")
+				.attr("class", "blob")
+				.attr("role", Roles.GRAPHICS_SYMBOL)
+				.attr("opacity", 0)
+				.attr("transform", `translate(${c.x}, ${c.y})`)
+				.attr("fill", group => colorScale(group.name))
+				.style("fill-opacity", configuration.opacity.selected)
+				.attr("stroke", group => colorScale(group.name))
+				.attr("d", group => oldRadialLineGenerator(group.data))
+				.call(selection => selection
+					.transition(this.services.transitions.getTransition("radar_blobs_enter", animate))
+					.attr("opacity", 1)
+					.attr("d", group => radialLineGenerator(group.data))
+				),
+			update => update
+				.call(selection => selection
+					.transition(this.services.transitions.getTransition("radar_blobs_update", animate))
+					.attr("opacity", 1)
+					.attr("transform", `translate(${c.x}, ${c.y})`)
+					.attr("d", group => radialLineGenerator(group.data))
+				),
+			exit => exit
+				.call(selection => selection
+					.transition(this.services.transitions.getTransition("radar_blobs_exit", animate))
+					.attr("d", group => radialLineGenerator(group.data))
+					.attr("opacity", 0)
+					.remove()
+				)
+		);
+
+		// data dots
+		const dots = DOMUtils.appendOrSelect(this.svg, "g.dots").attr("role", Roles.GROUP);
+		const dotsUpdate = dots.selectAll("circle").data(this.displayDataNormalized);
+		dotsUpdate.join(
+			enter => enter.append("circle").attr("role", Roles.GRAPHICS_SYMBOL),
+			update => update,
+			exit => exit.remove()
+		)
+		.attr("class", d => kebabCase(d.key))
+		.attr("cx", d => polarToCartesianCoords(xScale(d.key), yScale(d.value), c).x)
+		.attr("cy", d => polarToCartesianCoords(xScale(d.key), yScale(d.value), c).y)
+		.attr("r", 0)
+		.attr("opacity", 0)
+		.attr("fill", d => colorScale(d[this.groupMapsTo]));
+
+		// rectangles
+		const xAxesRect = DOMUtils.appendOrSelect(this.svg, "g.x-axes-rect").attr("role", Roles.GROUP);
+		const xAxisRectUpdate = xAxesRect.selectAll("rect").data(this.uniqKeys);
+		xAxisRectUpdate.join(
+			enter => enter.append("rect").attr("role", Roles.GRAPHICS_SYMBOL),
+			update => update,
+			exit => exit.remove()
+		)
+		.attr("x", c.x)
+		.attr("y", c.y - xAxisRectHeight / 2)
+		.attr("width", yScale.range()[1])
+		.attr("height", xAxisRectHeight)
+		.attr("fill", "red")
+		.style("fill-opacity", 0)
+		.attr("transform", key => `rotate(${radToDeg(xScale(key))}, ${c.x}, ${c.y})`);
+
+		// Add event listeners
+		this.addEventListeners();
+
+		oldYScale = yScale; // save the current scale as the old one
+	}
+
+	// append temporarily the label to get the exact space that it occupies
+	labelDimensions = (label: string) => {
+		const tmpTick = DOMUtils.appendOrSelect(this.svg, `g.tmp-tick`);
+		const tmpTickText = DOMUtils.appendOrSelect(tmpTick, `text`).text(label);
+		const { width, height } = DOMUtils.getSVGElementSize(tmpTickText.node(), { useBBox: true });
+		tmpTick.remove();
+		return { width, height };
```

Comment:
you can just return `DOMUtils.getSVGElementSize(tmpTickText.node(), { useBBox: true })`

Reply:
Mmmm I don't think so :) I need to calculate and save` { width, height }` before removing the node